src/main/java/com/frostcraft/admintools/FCAdminCommand.java

```diff
@@ -0,0 +1,532 @@
+package com.frostcraft.admintools;
```

Comment:
Any way we could make git pick this up as a rename to see the file diffs?

Reply:
Not that I know of.

Reply:
When compared manually, it is hard to determine the changes since I removed a redundent nil check (that would also be true) and re-indented most of the code.

Reply:
The changes I'm aware of is that I removed the deprecated warnings suppression, and updated the commands. The larges change would be that since item id is now deprecated, I changed it to item namespace.